services/cms/src/utils/Gutenberg/withMentimeterInspector.tsx

```diff
@@ -0,0 +1,84 @@
+import { css } from "@emotion/css"
+import { InspectorControls } from "@wordpress/block-editor"
+import { PanelBody, TextControl } from "@wordpress/components"
+import { createHigherOrderComponent } from "@wordpress/compose"
+import { Fragment } from "@wordpress/element"
+import { useTranslation } from "react-i18next"
+
+import { baseTheme } from "../../shared-module/styles"
+
+// https://developer.wordpress.org/block-editor/reference-guides/filters/block-filters/#editor-blockedit
+const withMentimeterInspector = createHigherOrderComponent((BlockEdit) => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const MentiMeterEmbed = (props: any) => {
+    const { t } = useTranslation()
+    if (props.attributes.providerNameSlug !== "mentimeter") {
+      return <BlockEdit {...props} />
+    }
+
+    function updateQueryStringParameter(url: string | undefined, key: string, value: string) {
+      if (!url) {
```

Comment:
Makes more sense to handle this case where this function is called. This function also could be moved to shared-module.